runtime/wasm/src/host_exports.rs

```diff
@@ -65,6 +65,34 @@ where
         }
     }
 
+    pub(crate) fn abort(
+        &self,
+        message: Option<String>,
+        file_name: Option<String>,
+        line_number: Option<u32>,
+        column_number: Option<u32>,
+    ) -> Result<(), HostExportError<impl ExportError>> {
+        let message = message
+            .map(|message| format!("message: `{}`", message))
```

Comment:
Can we format the message without backticks around it?

Reply:
Hmm maybe backticks aren't right here but I do like to mark it somehow, Rust backtraces use single quotes `'message'` for the panic message, would that be better?

Reply:
I feel the : already marks it clear enough.

Reply:
Done